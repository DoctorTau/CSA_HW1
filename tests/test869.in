281
-3232 -8626 -6929 9267 -9528 -1621 9635 6041 4416 -3195 -5278 -5684 3700 4168 6288 5451 9040 6055 -5989 8462 -8403 -3219 7429 2564 -3118 -7627 1204 9514 3841 -8376 -314 -726 -4490 2555 1757 -4205 5829 -1959 7967 6730 -8652 9543 -6420 -5995 4235 6042 7265 -8780 -3957 -7257 -882 2567 1945 8657 6829 -9530 -9929 -693 -6085 3228 -4069 -7991 7968 6771 -4061 6437 -424 5499 -7490 -7168 -2253 5160 143 -4056 5795 837 -4606 -1563 -498 5702 -4350 9852 1294 -5315 -8363 -1545 4730 8782 -5250 -4924 3174 1965 9297 -256 2331 9842 1415 6328 -5535 -6879 -5877 2625 -9695 -5700 -5841 -3086 5046 4369 9925 7832 -6194 6180 -4516 9589 1336 -5668 -1709 -1913 -700 9248 3173 -996 2829 -5002 6912 796 5455 9381 -1286 3227 3396 -45 -6660 1215 5145 1819 -7426 383 5378 -9746 -7219 -3876 -190 -6845 -6669 2593 -9163 8453 9980 -8855 7690 -5497 -2284 3380 -4279 -5242 8073 -3376 3693 5162 731 6721 9901 759 1996 -7712 6675 7684 -9185 -5912 4499 -5515 2634 -1046 3845 6315 -8432 9586 -2195 7903 2513 -580 9166 -5177 2036 5168 3263 5146 2218 3710 -3494 -2024 -468 2074 -2208 8596 3179 -7682 -5095 7101 4884 1448 -9214 2264 -6871 9986 1569 -6830 5874 -3995 4125 -4971 3423 -5309 -1162 5233 6939 5427 -1391 -7804 8602 -8075 -1487 5280 5980 -3230 -2621 1070 137 5914 -8115 3178 9571 3207 -1623 -7832 1102 5315 9755 1957 1079 4159 2919 3438 4317 -196 -3052 -7091 6506 -1936 7344 -9715 -420 -4648 9447 7330 -6617 6405 -1288 4480 8779 1627 -4224 -552 2693 -4676 4957 7926 9441 1978 9794 -7269 8186 1598 1175 4522 -9649 7471 5509 -8707 -8331 