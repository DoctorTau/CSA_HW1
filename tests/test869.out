-3232 -8626 -9528 -5278 -5684 -3118 -8376 -314 -726 -4490 -8652 -6420 -8780 -882 -9530 -424 -7490 -7168 -4056 -4606 -498 -4350 -5250 -4924 -256 -5700 -3086 -6194 -4516 -5668 -700 -996 -5002 -1286 -6660 -7426 -9746 -3876 -190 -2284 -5242 -3376 -7712 -5912 -1046 -8432 -580 -3494 -2024 -468 -2208 -7682 -9214 -6830 -1162 -7804 -3230 -7832 -196 -3052 -1936 -420 -4648 -1288 -4224 -552 -4676 